coffeecard/CoffeeCard.Library/Services/v2/TicketService.cs

```diff
@@ -49,5 +55,46 @@ public Task<List<TicketResponse>> GetTickets(User user, bool includeUsed)
                     ProductName = t.Purchase.ProductName
                 }).ToListAsync();
         }
+        
+        public async Task<TicketDto> UseTicket(int userId, int productId)
```

Comment:
We load the user in the comtroller, let's have user as the parameter and not the userId

Reply:
See comments from interface